src/components/ui.js

```diff
@@ -1,7 +1,7 @@
 /* globals AFRAME THREE */
 AFRAME.registerComponent('ui', {
   schema: { brightness: { default: 1.0, max: 1.0, min: 0.0 } },
-  dependencies: ['ui-raycaster'],
+  dependencies: ['ui-raycaster', 'screenshot-camera', 'brush'],
```

Comment:
i think i misunderstood dependencies.  Is `ui-raycaster` needed as a dependency because we modify it in the init() method?  

And are you thinking `screenshot-controls` would be different than the `screenshot-camera` component i added, or I should just rename it?